lib/connextclient/ConnextClient.ts

```diff
@@ -784,14 +939,16 @@ class ConnextClient extends SwapClient {
     }
 
     const withdrawResponse = await this.sendRequest('/withdraw', 'POST', {
-      recipient: destination,
+      publicIdentifier: this.publicIdentifier,
+      channelAddress: this.channelAddress,
       amount: amount.toLocaleString('fullwide', { useGrouping: false }),
       assetId: this.tokenAddresses.get(currency),
+      recipient: destination,
+      fee: '120', // TODO: estimate fee
```

Comment:
Can be done with a call to the eth provider: [eth_gasPrice](https://eth.wiki/json-rpc/API#eth_gasprice)

What is the default value when it is not set here?

Reply:
Thanks, will address this in a separate PR. Static "120" works for simnet.